slackbot/plugins/jira.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+from errbot import BotPlugin
+from errbot import botcmd
+from itertools import chain
+import requests
+import logging
+import json
+import re
+import os
+import random
+
+from slackbot.utils.secrets import Secrets
+
+secrets = Secrets()
+
+log = logging.getLogger(name='errbot.plugins.Jira')
+
+CONFIG_TEMPLATE = {'API_URL': secrets.get('Jira', 'JIRA_URL'),
+                   'USERNAME': secrets.get('Jira', 'JIRA_USERNAME'),
+                   'PASSWORD': secrets.get('Jira', 'JIRA_PASSWORD'),
+                   'OAUTH_ACCESS_TOKEN': None,
+                   'OAUTH_ACCESS_TOKEN_SECRET': None,
+                   'OAUTH_CONSUMER_KEY': None,
+                   'OAUTH_KEY_CERT_FILE': None,
+                   'FIELD_CHECK': "customfield_901103"}
+
+try:
```

Comment:
why do we need to try here?  shouldn't it already be installed in docker container?